README.md

```diff
@@ -1,8 +1,10 @@
 # project-sandbox
-a playground for experimentation
+Project Sandbox is a pilot program that helps us reimagine the relationship between a civic tech community and it's local government by putting open data platforms at the core of how work gets done. 
+
+We will use maps and Github to better understand how the City can use the properties it owns to drive economic development - specifically as this ties to Business Improvement Districts, City Council Districts, surplus land, and other areas we might find interesting. See definitions of these and other terms below. 
```

Comment:
We will use maps and <a href="https://github.com/about">Gthub</a> to better understand how the City can use the properties it owns to drive economic development. Github is an open (accessible to everyone) tool to share code and data with the public, manage tasks and deploy civic projects. 

Project Sandbox will shine a light specifically on the intersection of Business Improvement Districts, City Council Districts, surplus land, and other areas we might find interesting. See definitions of these and other terms below.